### DISCUSSION TITLE
Update eww without building from scratch

### DISCUSSION THREAD

**Discussion by mantacid** [elkowar/eww#882] [Ideas]
When I built eww the first time, it took forever. While this wouldn’t be a high-priority feature, it would be a nice QoL change to allow people to update eww instead of having to reinstall it.

Please tell me if this is already a thing and I’m just oblivious to it.

**Comment by elkowar** [comment]
What exactly would that look like? If you want to not have to rebuild everything from scratch, just... don't.
What's missing in this workflow?

## First build
```sh
$ git clone https://github.com/elkowar/eww
$ cd eww
$ cargo build --release
```
your binary is now in `eww/target/release/eww`, install however you wish to

## later, after an update
```sh
$ cd eww
$ git pull
$ cargo build --release
```
updated binary is now in `eww/target/release/eww`, install however you wish to

**Reply by mantacid** [reply]
Ahh, I see. It appears I am simply stupid. Thank you for the clarification. I was unaware you could even do such a thing with git.
